website/docs/components/code-block/partials/code/component-api.md

```diff
@@ -0,0 +1,44 @@
+## Component API
+
+This component uses [prism.js](https://prismjs.com/) under the hood.
+
+<Doc::ComponentApi as |C|>
```

Comment:
missing `onInput` ?

Reply:
It was indeed missing, now that we decided to remove the editable feature is not needed anymore.